Makefile

```diff
@@ -60,11 +65,21 @@ manager: generate fmt vet
 run: generate fmt vet manifests
 	go run ./main.go --metrics-addr=:8089
 
+$(GITREPO_CRD):
```

Comment:
If I run this once then Make would never download the CRD again, even if `SOURCE_VER` changes, would it? I'd have to run `cleanup-crd-deps` manually beforehand.